03_letters/brvishnu92/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := `a:2
+b:1
+`
+	actual := buf.String()
+	assert.Equalf(t, expected, actual, "got %v want %v", actual, expected)
+}
+
+func TestLetters(t *testing.T) {
+	testCases := map[string]struct {
+		input    string
+		expected map[rune]int
+	}{
+		"test1":   {input: "blahbla", expected: map[rune]int{97: 2, 98: 2, 104: 1, 108: 2}},
```

Comment:
These tests would all be a lot clearer if you used the rune as the key and not its integer value: `map[rune]int{'a': 2, 'b': 2, 'h': 1, 'l': 2}`

Reply:
Sure!! :-D Done.

Reply:
I've been waiting for you to push these changes before re-reviewing, but perhaps you just forgot.

You said "Done", but I don't see that it has been done. Please double-check your response to comments before asking for re-review. Thanks